Doc/library/gc.rst

```diff
@@ -68,13 +78,20 @@ The :mod:`gc` module provides the following functions:
 
 .. function:: get_objects(generation=None)
 
+
    Returns a list of all objects tracked by the collector, excluding the list
-   returned. If *generation* is not ``None``, return only the objects tracked by
-   the collector that are in that generation.
+   returned. If *generation* is not ``None``, return only the objects as follows:
+
+   * 0: All objects in the young generation
+   * 1: No objects, as there is no generation 1
```

Comment:
```suggestion
   * 1: No objects, as there is no generation 1 (as of Python 3.13)
```

Reply:
I know this information is below, but it just seems weird to say "Option A: There is no option A".  I'm sure there's a Holy Hand Grenade reference in there somewhere.